<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RandomFiberStraightCircle</class>
 <widget class="QDialog" name="RandomFiberStraightCircle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="randFiberGenerationsLabel">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 16pt &quot;Serif&quot;;</string>
           </property>
           <property name="text">
            <string>Random Fiber Generations</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="straightCirclesLabel">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Serif&quot;;</string>
           </property>
           <property name="text">
            <string>Straight Circles</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 11pt &quot;Serif&quot;;</string>
           </property>
           <property name="text">
            <string>Domain Dimensions</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="xSizeLabel">
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;Serif&quot;;</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::Panel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text">
              <string>x-size:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="xSizeEdit">
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;Serif&quot;;</string>
             </property>
             <property name="text">
              <string>200</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="ySizeLabel">
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;Serif&quot;;</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::Panel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text">
              <string>y-size:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ySizeEdit">
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;Serif&quot;;</string>
             </property>
             <property name="text">
              <string>200</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="zSizeLabel">
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;Serif&quot;;</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::Panel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text">
              <string>z-size:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="zSizeEdit">
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;Serif&quot;;</string>
             </property>
             <property name="text">
              <string>200</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 11pt &quot;Serif&quot;;</string>
           </property>
           <property name="text">
            <string>Fiber Length</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="avgLength">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;Serif&quot;;</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::Panel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text">
              <string>Average Length:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="avgLengthEdit">
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;Serif&quot;;</string>
             </property>
             <property name="text">
              <string>200</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="lengthDevLabel">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;Serif&quot;;</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::Panel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text">
              <string>Length Deviation:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lengthDevEdit">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="circleDimensionsLabel">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 11pt &quot;Serif&quot;;</string>
           </property>
           <property name="text">
            <string>Circle Dimensions</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="avgRadiusLabel">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;Serif&quot;;</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::Panel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text">
              <string>Average Radius:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="avgRadiusEdit">
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;Serif&quot;;</string>
             </property>
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="radiusDevLabel">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;Serif&quot;;</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::Panel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text">
              <string>Radius Deviation:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="radiusDevEdit">
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;Serif&quot;;</string>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="Intersect">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Allow Fibers to Intersect</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="fibAngleVarLabel">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Serif&quot;;</string>
         </property>
         <property name="text">
          <string>Fiber Angle Variability</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="angle_combo">
         <item>
          <property name="text">
           <string>Isotropic</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Transverse Isotropic</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1D</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="direction_label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Serif&quot;;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Panel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string>Direction:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="direction_combo">
           <property name="currentIndex">
            <number>2</number>
           </property>
           <item>
            <property name="text">
             <string>x</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>y</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>z</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="variability_label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Serif&quot;;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Panel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string>Variability (deg):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="variability_edit">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Serif&quot;;</string>
           </property>
           <property name="text">
            <string>15</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="ratioVoidVolume">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Serif&quot;;</string>
         </property>
         <property name="text">
          <string>Ratio of Void to Domain Volume </string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="porosityLabel">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Serif&quot;;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Panel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string>Porosity:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="porosityEdit">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Serif&quot;;</string>
           </property>
           <property name="text">
            <string>0.8</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="randSeedValuesLabel">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Serif&quot;;</string>
         </property>
         <property name="text">
          <string>Value from 1 to 2^15</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="randSeedLabel">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose a value between [1, 2^15]. This will be the starting position of the random generator.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Serif&quot;;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Panel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string>Random Seed:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="randSeedEdit">
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose a value between [1, 2^15]. This will be the starting position of the random generator.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Serif&quot;;</string>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QCheckBox" name="bindCheck">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Bind Fibers</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="avgRadiusLabel_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Serif&quot;;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Binder Radius:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="bindRadiusEdit">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Serif&quot;;</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>99</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Generate">
       <property name="minimumSize">
        <size>
         <width>175</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Serif&quot;;</string>
       </property>
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>xSizeEdit</tabstop>
  <tabstop>ySizeEdit</tabstop>
  <tabstop>zSizeEdit</tabstop>
  <tabstop>avgLengthEdit</tabstop>
  <tabstop>avgRadiusEdit</tabstop>
  <tabstop>lengthDevEdit</tabstop>
  <tabstop>radiusDevEdit</tabstop>
  <tabstop>Intersect</tabstop>
  <tabstop>porosityEdit</tabstop>
  <tabstop>randSeedEdit</tabstop>
  <tabstop>bindCheck</tabstop>
  <tabstop>bindRadiusEdit</tabstop>
  <tabstop>Generate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
